specification/compute/resource-manager/Microsoft.Compute/stable/2019-11-01/disk.json

```diff
@@ -0,0 +1,2036 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "DiskResourceProviderClient",
+    "description": "The Disk Resource Provider Client.",
+    "version": "2019-07-01"
```

Comment:
>2019-07-01 [](start = 16, length = 10)

This should be `2019-11-01` to match the directory name.

Reply:
All the examples as well.

---
In reply to: [369914444](https://github.com/Azure/azure-rest-api-specs/pull/8195#discussion_r369914444) [](ancestors = 369914444)

Reply:
Sorry, completely forgot to change these

Reply:
np

---
In reply to: [369917621](https://github.com/Azure/azure-rest-api-specs/pull/8195#discussion_r369917621) [](ancestors = 369917621)